gensim/models/keyedvectors.py

```diff
@@ -1035,7 +1096,7 @@ def rank_by_centrality(self, words, use_norm=True):
         if not used_words:
             raise ValueError("cannot select a word from an empty list")
         vectors = vstack([self.get_vector(word, norm=use_norm) for word in used_words]).astype(REAL)
-        mean = matutils.unitvec(vectors.mean(axis=0)).astype(REAL)
+        mean = matutils.unitvec(self.get_mean_vector(vectors)).astype(REAL)
```

Comment:
Another norming that might make sense to pull into a `get_mean_vector()` option.